csharp/Platform.Collections/Arrays/ArrayFiller[TElement, TReturnConstant].cs

```diff
@@ -1,29 +1,82 @@
 ﻿using System.Collections.Generic;
 using System.Runtime.CompilerServices;
 
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
-
 namespace Platform.Collections.Arrays
 {
+    /// <summary>
+    /// <para>Provides <see cref="T:TElement[]"/> array filler with additional methods that return a given constant of type <see cref="TReturnConstant"/>.</para>
+    /// <para>Предоставляет заполнитель массива <see cref="T:TElement[]"/> c дополнительными методами, возвращающими заданную константу типа <see cref="TReturnConstant"/>.</para> 
+    /// </summary>
+    /// <typeparam name="TElement"><para>The elements' type.</para><para>Тип элементов массива.</para></typeparam>
+    /// <typeparam name="TReturnConstant">The return constant' type.</typeparam>
     public class ArrayFiller<TElement, TReturnConstant> : ArrayFiller<TElement>
     {
         protected readonly TReturnConstant _returnConstant;
 
+        /// <summary>
+        /// <para>Initializes a new instance of the class <see cref="ArrayFiller"/> using the specified array, the offset from which filling will start and the constant returned when filling elements.</para>
+        /// <para>Инициализирует новый экземпляр класса <see cref="ArrayFiller"/> используя указанный массив, смещение с которого начнётся заполнение и константу возвращаемую при заполнении элементов.</para>
+        /// </summary>
+        /// <param name="array"><para>The array to fill.</para><para>Массив для заполнения.</para></param>
+        /// <param name="offset"><para>The offset from which to start filling the array.</para><para>Смещение с которого начнётся заполнение массива.</para></param>
+        /// <param name="returnConstant"><para>The constant's value.</para><para>Значение константы.</para></param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public ArrayFiller(TElement[] array, long offset, TReturnConstant returnConstant) : base(array, offset) => _returnConstant = returnConstant;
 
+        /// <summary>
+        /// <para>Initializes a new instance of the class <see cref="ArrayFiller"/> using the specified array and the constant returned when filling elements. Filling will start from the beginning of the array.</para>
```

Comment:
`the <see cref="ArrayFiller"/> class` instead of `the class <see cref="ArrayFiller"/>`